src/main/scala/rps/Main.scala

```diff
@@ -1,51 +1,83 @@
 package rps
 import scala.io.StdIn.readLine
 import scala.util.Random
-import io.buildo.enumero.annotations.indexedEnum
+import io.buildo.enumero.annotations.enum
 import io.buildo.enumero.{CaseEnumIndex, CaseEnumSerialization}
 import Move._
+import Result._
+import de.heikoseeberger.akkahttpcirce.ErrorAccumulatingCirceSupport._
+import io.circe.generic.auto._
+import io.buildo.enumero.circe._
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.server.Directives._
+import akka.stream.ActorMaterializer
+import scala.io.StdIn
 
 object Main extends App {
-  Game.play
+  WebServer.run
 }
 
 object Game {
-  def play() : Unit = {
-
-  println("Choose Rock (0),PAPER (1),SCISSOR (2)")
-  val move = CaseEnumIndex[Move].caseFromIndex(readLine())
-
-  val computerMove = generateRandom()
-  val output = move.map(m => {
-  val resultString = s"${CaseEnumSerialization[Move].caseToString(m)} VS ${CaseEnumSerialization[Move].caseToString(computerMove)}-> "
-  val result =(m, computerMove) match {
-        case (x,y) if x == y => "Draw!"
-        case (ROCK,PAPER) | (PAPER,SCISSOR) | (SCISSOR,ROCK) => "You lost!"
-        case (ROCK,SCISSOR) | (PAPER,ROCK) | (SCISSOR,PAPER)  => "You win!"
-        case _ => "Unknown combination!"
-      }    
-      s"$resultString $result"
-    }).getOrElse("Not an Option!")
-    println(output)
-  }
+  def play(move : Move) : (Move,Move,Result) = {
```

Comment:
Returning a Tuple can be confusing, I would rather declare a case class with this data and pass it around, so you the caller of this method is aware of what field is using